lib/gruf/server.rb

```diff
@@ -61,7 +61,9 @@ def server
     def start!
       logger.info { 'Booting gRPC Server...' }
       server.run_till_terminated
+    rescue Interrupt, SignalException, SystemExit
```

Comment:
Can you wrap this method in `# rubocop:disable Lint/ShadowedException` and `# rubocop:enable Lint/ShadowedException` flags? (May need to rebase this from master)

Reply:
👍  updated